meetings/2022-12-05.md

```diff
@@ -0,0 +1,91 @@
+* WT: I tried to mention / use the SAI vocab in here - so parts of SAI that are similar could be left out, and then decide whether to change vocab here to reflect original vocab. Don't think it's necessary to have general and more specific - just make one that's standalone and in rest of SAI extend with specific implementation of registration procedures
+* Pavlik: Currently in Sai the Authz Agent handles discovery of Agent Registration. It has logic based on client and end user identities to return the right registration. Would you see this part defined here or in SAI
+* WT: Could include here, or could be implementation direction
+* Pavlik: Discussing possibility of having a special public agent registration and IRI for public agent (I think ACP does this). If we have something like that - maybe there would be two link headers returned for public and agent specific - or add a special triple to shortcut the hub - because you won't need to request authorization for public agent
+* WT: If you're a public agent and you act as such - and are registered nowhere - then you could nevertheless pull from the RDH to get the public agent location, or could use a specific triple in the webid. But using this mechanism you can separate technical stuff from the webid and leave it open for read/write of profile or whatever else you want to do with it.
+* Pavlik: I wouldn't say the client is acting as a public agent - any agent is always acting as a public agent plus an authorized agent. So you can get agent specific one plus public one. If you wanted one entry point it would be useful to always get the public registration as a separate link in a header. Advertising in a webid document could be seen as a shortcut (nice-to-have). Could also have a different predicate to advertise it. Would still have a structure of agent registration. To create it you'd still want to use an authorization agent.
+* Pavlik: In general I think it's an interesting approach to extract something but need to see how to recombine it. Definitely worth exploring.
+* Justin: I agree. One of the criticisms is that is to big, and it has gotten in the way of adoption. Having it more modular and havig pieces that can be adopted one at the time. I think part of creating somthing like this must to be paired with what changes will need to be done in the spec document.
+* ...: The current spec is imlemented so we know it works, if we extract one and we update the main document to complement it we should know that it will stil work.
+* ...: I think we still should mention how to use type index or migrate from it. Showing the pathway or a bride is important.
+* WT: I think so and this is the main reason I started doing this. The quesiton is very importan about what is this Public Agent Registration. Do we envision it as something separate again? Or do we say that this could be a document pointed by a discovery hub, but is not unimaginable that it would be the webid itself.
```

Comment:
```suggestion
* WT: I think so, and this is the main reason I started doing this. The very important question is, what is this Public Agent Registration? Do we envision it as something separate again? Or do we say that this could be a document pointed to by a discovery hub, but is not unimaginable that it would be the WebID itself?
```